app/retail/templates/settings/account.html

```diff
@@ -87,16 +87,16 @@
     padding-right: 40px;
   }
 
-  #export-modal-label {
+  #exportModalLabel {
```

Comment:
Is bootstrap loaded on these pages? If it is, we should use those styles here instead.

Reply:
Bootstrap is loaded, but I found I needed to do a lot of styling to implement the requested design. I'll look in to which styles I can replace with bootstrap classes though. This was just the quickest way at the time.

Reply:
@Jer-Sch In general using the css library is much better than hard-coded values since it provides consistency across the application. Since this page is likely going to be redesigned soon, it's not worth spending more time on it. We can leave these values in and when this is Vue-ified these styles can be updated during that effort.